### ISSUE TITLE
TEAC USB Floppy drive head bangs with no disk inserted

### ISSUE THREAD

**Issue by mike-cuche** [storaged-project/udisks#807]
Hello there,
I've got a Teac UF000x usb floppy drive (ID 0644:0000) which works fine when a floppy is inserted, but when is not every second the head bangs and the led flashes like it's constantly trying to check if a floppy is inserted into the drive; which is annoying and it may damage the head over the time.
This does not happen on Windows, the drive stays in idle and activates only when effectively accessed

I've already tried to search the interweb to figure out solving this issue with no luck, I wonder it's related to udisks somehow

**Comment by vpodzime** [comment]
> I wonder it's related to udisks somehow

Did you try stopping udisks? (ideally with `systemctl mask`)

**Comment by mike-cuche** [comment]
Yes, I've tried right now but nothing changed, the drive still clicks

**Comment by vpodzime** [comment]
Not related to udisks then, I'm afraid. I guess you checked `journalctl` and `dmesg`?

**Comment by mike-cuche** [comment]
It was the first thing i did, journalctl shows only dmesg messages when the drive is plugged in, but it nothing appears when the drives click.
I guess I'm done on this thread, so I'm closing it and search anywhere else to report.
